ports/atmel-samd/common-hal/pwmio/PWMOut.c

```diff
@@ -52,17 +52,13 @@ uint8_t tcc_refcount[TCC_INST_NUM];
 
 // This bitmask keeps track of which channels of a TCC are currently claimed.
 #ifdef SAMD21
-uint8_t tcc_channels[3];   // Set by pwmout_reset() to {0xf0, 0xfc, 0xfc} initially.
+uint8_t tcc_channels[3];
 #endif
 #ifdef SAM_D5X_E5X
-uint8_t tcc_channels[5];   // Set by pwmout_reset() to {0xc0, 0xf0, 0xf8, 0xfc, 0xfc} initially.
+uint8_t tcc_channels[5];
 #endif
 
 
-void common_hal_pwmio_pwmout_never_reset(pwmio_pwmout_obj_t *self) {
-    never_reset_pin_number(self->pin->number);
```

Comment:
This is why we still need this. Or at least, we need to replace the pwm never_reset with pin never_reset.